From 676ce92febebbae76a8860e7299c9b157dbb73d2 Mon Sep 17 00:00:00 2001
From: Stephane Glondu <steph@glondu.net>
Date: Fri, 9 Aug 2019 07:16:46 +0200
Subject: [PATCH] Fix a leaking open file (Closes: #933827)

The file debian/source/format was open by "dgit fetch" but never
closed, causing failure to remove the extracted tree on NFS.
---
 dgit | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dgit b/dgit
index 6401524e..b2147244 100755
--- a/dgit
+++ b/dgit
@@ -4020,6 +4020,7 @@ sub get_source_format () {
     }
     $_ = <F>;
     F->error and confess "$!";
+    close F;
     chomp;
     return ($_, \%options);
 }
-- 
2.20.1

